Src/Particle/AMReX_ParticleCommunication.H

```diff
@@ -467,7 +467,7 @@ void communicateParticlesStart (const PC& pc, ParticleCopyPlan& plan, const Buff
         const auto offset = rOffset[i];
         Long nbytes       = plan.m_rcv_num_particles[Who]*psize;
         std::size_t acd   = ParallelDescriptor::alignof_comm_data(nbytes);
```

Comment:
This may trigger an unused variable warning because later `acd` is only used in `AMREX_ASSERT`.

Reply:
I think it's okay because it's still used in the line below.

Reply:
Oh, yes.  How did I miss it?